giskard/llm/config.py

```diff
@@ -43,5 +44,29 @@ def set_default_llm(self, default_llm=None):
 
         self._default_llm = default_llm
 
+    def build_scan_llm(self, gpt4_preferred: bool = False, **kwargs):
+        from langchain.chat_models import ChatOpenAI
+
+        if "OPENAI_API_KEY" not in os.environ:
+            raise EnvironmentError(
+                """
+                You're trying to use giskard LLM scanning features without providing any LLM or OpenAI API key.
+                To use the default OpenAI API please provide the OPENAI_API_KEY environment variable.
+                """
+            )
+
+        model = (
+            "gpt-3.5-turbo"
```

Comment:
General comment about model names: I wonder if we should move them to Settings to make them configurable from env variables or at least turn them into constants to avoid typos

Reply:
We should just enforce our own model, it’s exclusively for internal usage. At least for now, I would hard code the model and config. We can add more configurable options (e.g. running on Azure) later.